lib/client-assets.php

```diff
@@ -218,116 +263,74 @@ function gutenberg_register_packages_scripts() {
 		);
 	}
 }
+add_action( 'wp_default_scripts', 'gutenberg_register_packages_scripts' );
 
 /**
- * Registers common scripts and styles to be used as dependencies of the editor
- * and plugins.
+ * Registers all the WordPress packages styles that are in the standardized
+ * `build/` location.
  *
- * @since 0.1.0
- */
-function gutenberg_register_scripts_and_styles() {
-	global $wp_scripts;
-
-	gutenberg_register_vendor_scripts();
-	gutenberg_register_packages_scripts();
-
-	// Add nonce middleware which accounts for the absence of the heartbeat
-	// listener. This relies on API Fetch implementation running middlewares in
-	// order of last added, and that the original nonce middleware would defer
-	// to an X-WP-Nonce header already being present. This inline script should
-	// be removed once the following Core ticket is resolved in assigning the
-	// nonce received from heartbeat to the created middleware.
-	//
-	// See: https://core.trac.wordpress.org/ticket/46107 .
-	// See: https://github.com/WordPress/gutenberg/pull/13451 .
-	if ( isset( $wp_scripts->registered['wp-api-fetch'] ) ) {
-		$wp_scripts->registered['wp-api-fetch']->deps[] = 'wp-hooks';
-	}
-	wp_add_inline_script(
-		'wp-api-fetch',
-		sprintf(
-			'wp.apiFetch.nonceMiddleware = wp.apiFetch.createNonceMiddleware( "%s" );' .
-			'wp.apiFetch.use( wp.apiFetch.nonceMiddleware );' .
-			'wp.apiFetch.nonceEndpoint = "%s";',
-			( wp_installing() && ! is_multisite() ) ? '' : wp_create_nonce( 'wp_rest' ),
-			admin_url( 'admin-ajax.php?action=gutenberg_rest_nonce' )
-		),
-		'after'
-	);
-
-	// TEMPORARY: Core does not (yet) provide persistence migration from the
-	// introduction of the block editor and still calls the data plugins.
-	// We unset the existing inline scripts first.
-	$wp_scripts->registered['wp-data']->extra['after'] = array();
-	wp_add_inline_script(
-		'wp-data',
-		implode(
-			"\n",
-			array(
-				'( function() {',
-				'	var userId = ' . get_current_user_ID() . ';',
-				'	var storageKey = "WP_DATA_USER_" + userId;',
-				'	wp.data',
-				'		.use( wp.data.plugins.persistence, { storageKey: storageKey } );',
-				'	wp.data.plugins.persistence.__unstableMigrate( { storageKey: storageKey } );',
-				'} )();',
-			)
-		)
-	);
+ * @since 6.7.0
 
+ * @param WP_Styles $styles WP_Styles instance (passed by reference).
+ */
+function gutenberg_register_packages_styles( &$styles ) {
 	// Editor Styles.
-	// This empty stylesheet is defined to ensure backward compatibility.
-	gutenberg_override_style( 'wp-blocks', false );
```

Comment:
@youknowriad - do we still need this override? I couldn't find it in the WordPress core.

Reply:
yes, I think it's safe to remove

Reply:
I can't see it in this file:

https://github.com/WordPress/wordpress-develop/blob/ed1a53588f66b334136ab5cf2472745dedf8513f/src/wp-includes/script-loader.php

Searching in the repository confirms it as well:
https://github.com/WordPress/wordpress-develop/search?q=wp-blocks&unscoped_q=wp-blocks

Unless there is some magic method to do it...